src/Truncate/utils.js

```diff
@@ -0,0 +1,55 @@
+const LINE_HEIGHT_VALUE = 40;
+const CROP_DECREMENT_STEP = 0.01;
+
+const createCopyElement = (element) => {
+  const newElement = document.createElement(element.tagName);
+  newElement.setAttribute(
+    'style',
+    `line-height: ${LINE_HEIGHT_VALUE}px; display: inline-block;`,
+  );
+  return newElement;
+};
+
+const constructString = (text, whiteSpace, ellipsis) => {
+  const spacer = whiteSpace ? ' ' : '';
+  return `${text.trim()}${spacer}${ellipsis}`;
+};
+
+const cropText = (text, cropDecrement) => {
+  const sliceIndex = Math.floor(text.length * cropDecrement);
+  return text.slice(0, sliceIndex);
+};
+
+const truncateLines = (text, element, { lines, whiteSpace, ellipsis }) => {
```

Comment:
It might be good to add a JSDoc here to describe at a high level what this function does for future us/contributors.

Reply:
Good, done